.github/workflows/push.yaml

```
@@ -0,0 +1,262 @@
+          dart analyze
+          flutter pub global activate dartdoc
+          flutter pub global run dartdoc . --output talawa-mobile-docs --format md --exclude=test/widget_tests/widgets/pinned_carousel_widget_test.dart, lib/widgets/pinned_carousel_widget.dart, lib/widgets/post_widget.dart, test/widget_tests/widgets/post_widget_test.dart
+          rm -rf talawa-mobile-docs/widgets_pinned_carousel_widget/CustomCarouselScrollerState/build.md
+          rm -rf talawa-mobile-docs/widgets_post_widget/PostContainerState/build.md
+      - uses: actions/upload-artifact@v1
+        with:
+          name: talawa-mobile-docs
+          path: talawa-mobile-docs
+      - name: Checking doc updated
+        id: DocUpdated
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:28: Double quote to prevent globbing and word splitting

(shellcheck)